example-app/ios/ExampleApp.xcodeproj/xcshareddata/xcschemes/ExampleApp.xcscheme

```diff
@@ -60,6 +60,12 @@
             ReferencedContainer = "container:ExampleApp.xcodeproj">
          </BuildableReference>
       </BuildableProductRunnable>
+      <CommandLineArguments>
+         <CommandLineArgument
+            argument = ""
+            isEnabled = "YES">
+         </CommandLineArgument>
+      </CommandLineArguments>
```

Comment:
What is this?

Reply:
It was added by xcode or the like when I added .env, I'll test to remove it and see if it was added when I was clicking around in xcode

Reply:
Removed